image/docker_schema2.go

```diff
@@ -197,7 +208,7 @@ func (m *manifestSchema2) convertToManifestOCI1() (types.Image, error) {
 	// Rather than copying the ConfigBlob now, we just pass m.src to the
 	// translated manifest, since the only difference is the mediatype of
 	// descriptors there is no change to any blob stored in m.src.
```

Comment:
This comment is no longer true. (Passing `m.src` is fine, just don’t say that we are not copying the `ConfigBlob`.)